src/RadialGauge/RadialGaugeSeries/RadialGaugeSeries.tsx

```diff
@@ -1,17 +1,21 @@
-import React, { Component, Fragment, ReactElement } from 'react';
+import React, { cloneElement, ReactElement } from 'react';
+
+import { range, min } from 'd3-array';
+import { scaleBand } from 'd3-scale';
+
 import { ChartShallowDataShape } from '../../common/data';
-import { CloneElement } from 'rdk';
-import { RadialGaugeArcProps, RadialGaugeArc } from './RadialGaugeArc';
-import { RadialGaugeLabel, RadialGaugeLabelProps } from './RadialGaugeLabel';
+import { ColorSchemeType, getColor } from '../../common/color';
+
 import {
+  RadialGaugeArc,
+  RadialGaugeArcProps,
+  RadialGaugeLabel,
+  RadialGaugeLabelProps,
   RadialGaugeValueLabel,
   RadialGaugeValueLabelProps
-} from './RadialGaugeValueLabel';
-import { getColor, ColorSchemeType } from '../../common/color';
-import { range, min } from 'd3-array';
-import { scaleBand } from 'd3-scale';
+} from '.';
```

Comment:
I tend to avoid this since it can create circulars in some scenarios.

Reply:
replaced by import directly from files